### PR TITLE
Moderate shoes update

### PR DESCRIPTION
**Pull request by Kaedwuff**
All cloth and fabric footwear, including those ragged mystic cloth boots cultists wear, is now silent.  Sneak up on people with bunny slippers!  

Chameleon shoes are also now silent, because it fits with their stealth thematic.

White shoes were 99% chemical resistant because no one could be bothered to make a pair of shoes that were specifically for work with chemicals instead of designating all white shoes as doctor shoes.  There is now a separate chem resistant pair of shoes distinctive from shoes that just happen to be white.

### CODE REVIEW

File: html/changelogs/Kaedwuff - Shoes.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - tweak: "All soft and cloth footwear no longer clicks on the floor as you move. Certainy spooky or traitorous footwear is also affected by this. You may now stalk people in bunny slippers."
+  - tweak: "After a lawsuit involving accusations of carcinogenics used in the manufacturing process, all white shoes have been recalled.  The new brand of white shoes is no longer resistant to chemical spills, and are made of 0% asbestos."
```
Comment (Arrow768) [inline html/changelogs/Kaedwuff - Shoes.yml]: That is something for ic news

**Comment by alsoandanswer** [comment]
im pretty sure white shoes had that chem resistance because they are part of the chemist's loadout, and by extension, science and medical

**Comment by Kaedwuff** [comment]
Further updates have been made to reflect suggestions here.

File: html/changelogs/Kaedwuff - Shoes.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - tweak: "All soft and cloth footwear no longer clicks on the floor as you move. Certainy spooky or traitorous footwear is also affected by this. You may now stalk people in bunny slippers."
```
Comment (Arrow768) [inline html/changelogs/Kaedwuff - Shoes.yml]:
```suggestion
  - tweak: "All soft and cloth footwear no longer clicks on the floor as you move. Certain spooky or traitorous footwear is also affected by this. You may now stalk people in bunny slippers."
```

File: html/changelogs/Kaedwuff - Shoes.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  - tweak: "White shoes are no longer chemical resistant. Doctors and scientists are now equipped with special chem resistant shoes that are chem resistant, and distinctive from regular white shoes."
```
Comment (Arrow768) [inline html/changelogs/Kaedwuff - Shoes.yml]: Should be worded better than "special chem resistant shoes, that are chem resistant"
I.e.
```suggestion
  - tweak: "White shoes are no longer chemical resistant. Doctors and scientists are now equipped with special chem resistant shoes which are distinctive from regular white shoes."
```

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 4]
```diff
@@ -1,225 +1,229 @@
...
-	icon_state = "blackcaligae"
-	item_state = "blackcaligae"
-
-/obj/item/clothing/shoes/caligae/armor
-	name = "leather caligae"
-	desc = "The standard Unathi marching footwear. These are made for heavier conditions, featuring tough and waterproof eel-leather covering, offering far greater protection."
-	description_fluff = "These traditional Unathi footwear have remained relatively unchanged in principle, with improved materials and construction being the only notable change. This pair is reinforced with leather of the Zazehal, a Moghesian species of eel that can grow up to twenty five feet long. Typically, Zazehal Festivals are thrown every month of the warm season in which Unathi strew freshly killed birds across the shoreline and collect these creatures with baskets. The fungi that grow on their skin is harvested and used as an exotic seasoning, and their skin is used for its' incredibly durable, shark-like leather."
-	icon_state = "eelcaligae"
-	armor = list(melee = 40, bullet = 0, laser = 0, energy = 15, bomb = 20, bio = 0, rad = 20)
+/obj/item/clothing/shoes/syndigaloshes
```
Comment (Arrow768) [inline code/modules/clothing/shoes/miscellaneous.dm]: You most likely changed the line endings in that file

File: code/game/jobs/job/science.dm [hunk 5]
```diff
@@ -65,7 +65,7 @@
 
 	uniform = /obj/item/clothing/under/rank/scientist
 	suit = /obj/item/clothing/suit/storage/toggle/labcoat/science
-	shoes = /obj/item/clothing/shoes/white
+	shoes = /obj/item/clothing/shoes/medical
```
Comment (Arrow768) [inline code/game/jobs/job/science.dm]: You are reusing the medical shoes in science.
Create a stand alone type for that or change the path.

Comment (Kaedwuff) [reply code/game/jobs/job/science.dm]: @_@ they're all the same chem resistant shoes even if in the code they're tagged does Medical. They don't even have the word Medical in their name in Game. You want me to create four sets of exactly identical in appearance white shoes that all do something slightly different for each department?

Comment (Arrow768) [reply code/game/jobs/job/science.dm]: As I mentioned in my review:

> Create a shoe type for the scientists that has spill resistance
> Create a shoe type for the doctors that has bio resistance (bio armor)
> Create a shoe type for the viro/chem that has bio and spill resistance

**Comment by Kaedwuff** [comment]
Per arrows requests and with the help of smallgreenant, new shoes and sprites for those shoes have been added.

**Pull request merged**